### PR TITLE
Proposal to rephrase abstract and intro

### PR DESCRIPTION
**Pull request by mirjak**
Probably based on the history of this document, I think that the abstract and intro (at least) is too much focused on the bad sides of the robustness principle than the actual good recommends that this document which therefore might get easily lost for the reader. This PR is an attempt to refocus the scope of the abstract and intro to what the document actually wants to say or I believe it wants to say. Please take this as proposal for consideration and of course feel free to edit as you want!

Btw. I think the document is generally a bit the wrong way around. What I learnt about paper writing is that the important message should come first and the explanation follows - this is true on a per section, as well per paragraph, and per sentence level, e.g. in a section the first paragraph has the most important message and in each paragraph the first sentence contains the actual point and all other sentences are only for "remarks", i.e. ideally you can quick read a section by only reading the first sentence of each paragraph. Doing it the other ways around distracts the reader form the points you actually want to make, which I think is part of the problem here.

### CODE REVIEW

**Review by martinthomson — COMMENTED** [review]
Good idea to restructure this slightly.

The introductory text was written in 2015.  As you say, it could be more pointed.

I'll need to think about the specific suggestions here.  I'll need a quiet moment for that because on first reading there are some things that don't sit quite right.  Thanks for putting this out there; it is so much easier to deal with feedback when - like this - it has concrete suggestions in addition to the criticisms.

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -50,20 +50,43 @@ informative:
...
-The robustness principle, often phrased as "be conservative in what you send,
-and liberal in what you accept", has long guided the design and implementation
-of Internet protocols.  The posture this statement advocates promotes
-interoperability in the short term, but can negatively affect the protocol
-ecosystem over time.  For a protocol that is actively maintained, the robustness
-principle can, and should, be avoided.
+This document argues that active maintenance of protocols as well as
+protocol implementations is foremost important for interoperability. Active
+maintenance aims to detect and resolve interoperability problems quickly and
+as such retains flexibility and supports evolvability of the protocol.
```
Comment (mirjak) [inline draft-iab-protocol-maintenance.md:56]:
```suggestion
as such retains flexibility and supports evolvability of the protocol.
```

File: draft-iab-protocol-maintenance.md [hunk 2]
```diff
@@ -50,20 +50,43 @@ informative:
...
-interoperability in the short term, but can negatively affect the protocol
-ecosystem over time.  For a protocol that is actively maintained, the robustness
-principle can, and should, be avoided.
+This document argues that active maintenance of protocols as well as
+protocol implementations is foremost important for interoperability. Active
+maintenance aims to detect and resolve interoperability problems quickly and
+as such retains flexibility and supports evolvability of the protocol. 
+In contrast, the robustness principle, often phrased as "be conservative in what you send,
+and liberal in what you accept", promotes
+interoperability by bypassing failure, e.g. due to limitation in the specification or
```
Comment (mirjak) [inline draft-iab-protocol-maintenance.md:59]:
```suggestion
interoperability by bypassing failure, e.g. due to limitation in the specification or
```

File: draft-iab-protocol-maintenance.md [hunk 3]
```diff
@@ -50,20 +50,43 @@ informative:
...
 
 --- middle
 
 # Introduction
 
-The robustness principle has been hugely influential in shaping the design of
-the Internet. As stated in the IAB document on Architectural Principles of the
+There is good evidence to suggest that many important protocols are routinely
+maintained beyond their inception.  In particular, a sizeable proportion of IETF
+activity is dedicated to the stewardship of existing protocols.
```
Comment (mirjak) [inline draft-iab-protocol-maintenance.md:73]:
```suggestion
activity is dedicated to the stewardship of existing protocols.
```

**Pull request closed without merging**